controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -190,8 +190,10 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(ctx context.Context
 	nmstateOutput, err := nmstate.ApplyDesiredState(r.APIClient, instance.Spec.DesiredState)
 	if err != nil {
 		errmsg := fmt.Errorf("error reconciling NodeNetworkConfigurationPolicy at desired state apply: %s, %v", nmstateOutput, err)
+		encodedMessage, _ := encodeMessage(err.Error())
```

Comment:
Question: shouldn't this be:
```
encodedMessage, _ := encodeMessage(errmsg)
```

Reply:
This code was slightly modified and moved in the last commit into `SetFailedToConfigure` function in `pkg/enactmentstatus/conditions/conditions.go` and it works as expected (it is encoded also with the introductory string)